src/connection.cc

```diff
@@ -85,6 +100,34 @@ bool Connection::handle_read(const boost::system::error_code& error,
   return true; // success
 }
 
+std::string Connection::shortenPath(std::string path) {
```

Comment:
consider changing the name to `shortenUriPrefix` and the parameter to `uri_prefix`

side note: since the instructors use camelcase with first character capitalized like `FooBarFun` for function names, should we change all of our function names to that style? right now the function name style is all over the place